apps/agent/app/src/lib/utils.js

```diff
@@ -1,8 +1,13 @@
 import { round } from './math-utils'
+import { LOCALE_US_FORMAT } from './date-utils'
+
+export const ROUNDING_AMOUNT = 5
+
+export function noop() {}
 
 export function formatDecimals(value, digits) {
   try {
-    return value.toLocaleString('latn', {
+    return value.toLocaleString(LOCALE_US_FORMAT, {
```

Comment:
Is there a difference between `latn` and `en-US`?

Reply:
`latn` seems to not be supported by Firefox.